.github/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
+
+on: pull_request
+
+env:

```

Comment:
- [ ] [REQUIRED] Please, make sure to put the `linters.yml` file inside of `.github/workflows`. Yours is under `.github` directly.